lib/Core/ExecutorUtil.cpp

```diff
@@ -90,34 +97,47 @@ namespace klee {
 	const size_t numOperands = cv->getNumOperands();
 	kids.reserve(numOperands);
 	for (unsigned i = 0; i < numOperands; ++i) {
-	  kids.push_back(evalConstant(cv->getOperand(i)));
+	  kids.push_back(evalConstant(cv->getOperand(i), ki));
 	}
 	ref<Expr> res = ConcatExpr::createN(numOperands, kids.data());
 	assert(isa<ConstantExpr>(res) &&
 	       "result of constant vector built is not a constant");
 	return cast<ConstantExpr>(res);
       } else {
-	llvm::report_fatal_error("invalid argument to evalConstant()");
+	llvm::errs() << "ERROR: cannot handle constant '"
```

Comment:
Because there's currently no way to pass the location information to `klee_error`, but I think it's a good idea to add a function for this in `KInstruction`